src/graph/immutable_graph.cc

```diff
@@ -550,7 +550,7 @@ ImmutableGraph::EdgeArray ImmutableGraph::Edges(const std::string &order) const
     }
     std::copy(out_csr->indices.begin(), out_csr->indices.end(), rst_dst_data);
     std::copy(out_csr->edge_ids.begin(), out_csr->edge_ids.end(), rst_eid_data);
-  } else if (order == "eid") {
+  } else if (order.empty() || order == "eid") {
```

Comment:
Why change this condition?

Reply:
i think it causes the bug mentioned in #436

Reply:
I c